fakeiati.py

```diff
@@ -61,15 +61,16 @@ def generate_csv_suitecrm(corpus):
         fh.write(
             '"Name","ID","Website","Description","Date Created","Short Name","Data Portal URL",'
             '"Reporting Source Type","HQ Country","Default Publishing Licence","IATI Organisation Type",'
-            '"IATI Identifier","Address","Exclusions Policy URL","Email Address","Office Phone","Fax"\n'
+            '"IATI Identifier","Address","Exclusions Policy URL","Email Address",'
+            '"Office Phone","Fax","Approved to publish"\n'
         )
         for id in corpus.orgs:
             org = corpus.orgs[id]
-            fh.write(f'"{org.name}",{id},{org.url},"{org.description}",{org.created},')
-            fh.write(f'"{org.short_name}",{org.ui_url},{org.source_type},{org.country},')
-            fh.write(f"{org.default_license_id},{org.org_type},{org.iati_id},")
-            fh.write(f'"{org.address}",{org.exclusions_url},{org.contact_email},')
-            fh.write(f"{org.phone},{org.fax}\n")
+            created = org.created.strftime("%Y-%m-%d %H:%M:%S")
+            fh.write(f'"{org.name}",{id},{org.url},"{org.description}",{created},{org.short_name},{org.ui_url},')
```

Comment:
For similar things in the future, the `csv` module from the Standard Library could make life a bit easier because it avoids the need to do manual escaping and can write out rows direct from arrays or dictionaries.